### DISCUSSION TITLE
film box download issue

### DISCUSSION THREAD

**Discussion by CarolHuangnnn** [videovillage/community#188] [Q&A]
Hello, 
I’m downloading Filmbox on my Mac. But it paused. How can I solve this problem?
![S__170369027](https://github.com/videovillage/community/assets/170066310/9b659c2e-e10f-4b7c-9b1e-e362230d69dc)

**Comment by CarolHuangnnn** [comment]
![Uploading S__170377219.jpg…]()

**Comment by wilg** [comment]
Hi @CarolHuangnnn, please ensure your internet is stable, seems like that's the most likely cause. If you're still having trouble, please email support@videovillage.com so we can help you a bit more easily.

**Comment by CarolHuangnnn** [comment]
Thanks,
I installed the app successfully.

Wil Gieseler ***@***.***>於 2024年5月17日 週五，下午1:05寫道：
